website/docs/r/monitor_scheduled_query_rules_alert.html.markdown

```diff
@@ -25,16 +25,43 @@ resource "azurerm_application_insights" "example" {
   application_type    = "web"
 }
 
-resource "azurerm_log_analytics_workspace" "example" {
-  name                = "loganalytics"
+resource "azurerm_application_insights" "example2" {
+  name                = "appinsights2"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.example.name
+  application_type    = "web"
+}
+
+# Example: Alerting Action with result count trigger
+# Alert if more than three HTTP requests returned a >= 500 result code
```

Comment:
I don't think this is going to look quite right on the website. I'd throw a shorter description above the query attribute to explain what's going on

Reply:
Let me know if this latest changeset is more aligned with your thoughts.  It's looking a lot nicer to me.